enhancements/run-bundle.md

```diff
@@ -0,0 +1,263 @@
+---
+title: run-bundle
+authors:
+  - "@jmrodri"
+reviewers:
+  - "@joelanford"
+  - "@estroz"
+  - "@kevinrizza"
+  - "@ecordell"
+  - "@shawn-hurley"
+approvers:
+  - "@joelanford"
+  - "@estroz"
+  - "@ecordell"
+  - "@shawn-hurley"
+creation-date: 2020-05-07
+last-updated: 2020-05-07
+status: implementable
+see-also:
+replaces:
+superseded-by:
+
+---
+
+# Run Bundle
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+1. We currently check OLM installation status with `operator-sdk olm status`.
+   Can we change that to using the discovery API?
+   * Based on discussions, using the discovery API is the way to go.
+
+2. How do we handle --mode= flag on registry add
+   * should we ALWAYS default to --mode=semver?
```

Comment:
Should this be just "default", or does this mean "do not allow user to configure"?